frame/registry/README.md

```diff
@@ -0,0 +1,82 @@
+ # Liberland Registry Pallet
+
+ ## Overview
+
+ Registry pallet is a general-purpose pallet for tracking and registering
+ data about abstract entities.
+
+ In Liberland, it's used to implement Company Registry.
+
+ ## Terminology
+
+ * Entity - object, identified with AccountId, that has data attached to it and can be registered
+   at a registrar
+ * Registrar - AccountId that can register Entities in Registry.
+ * Registry - database of Entities and their data that were registered.
+ * Deposit - amount of Currency that gets reserved when requesting registration - refunded data
+   is removed
+
+ ## Entity Lifecycle
+
+ 1. Entity requests registration at Registry using `request_registration()` call. This creates
+    and stores a Registration Request and reserves the deposit.
+ 2. If Registrar approves Entity's data, they call `register_entity()` which
+    moves data from Registration Request to the Registry.
+ 3. To update the data, Entity needs to repeat the same process as for initial registration.
+ 3. Entity can be removed from Registry by the Registrar - deposit will be refunded.
+
+ ## Deposits
+
+ Registry pallet requires deposits to cover the cost of storing data in
+ current blockchain state (which isn't pruned). These deposits always matches
+ the amount of stored data:
+ * if there's no prior deposit, total required deposit will be reserved
+ * if there's a deposit, but now we want to store more data, the additional difference will be
+   reserved
+ * if there's a deposit, but now we want to store less or no data, the excess will be refunded
+
+ Formula for required deposit to store `N` bytes of data:
+ ```ignore
```

Comment:
Otherwise cargo treats it as a snippet of Rust code that should be tested. As this obviously isn't a valid Rust code, it fails, breaking the test suite. Read more about doc tests here: https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html

Strictly speaking I think it isn't needed here, as doc-tests are only taken from `*.rs` files, but this README.md is just directly extracted from the crate documentation (so from `lib.rs`)